rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/services/ServiceConfig.java

```diff
@@ -0,0 +1,35 @@
+package com.netflix.spinnaker.rosco.services;
```

Comment:
Do we need the standard copyright text at the top of this and other new classes?

Reply:
Good catch, looks like my intellij hasn't been configured for this project yet